### DISCUSSION TITLE
GitHub Copilot subscription has ended

### DISCUSSION THREAD

**Discussion by zjx00zjxlive** [community/community#113633] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I subscribed to the GitHub Copilot annual service on January 25, 2024, but now  it shows "Your GitHub Copilot subscription has ended." I'm not sure why this is happening.

**Comment by Flutterboy17** [comment]
Me too facing this problem after 6:00am today....

**Comment by ayim** [comment]
@zjx00zjxlive @Flutterboy17 which IDE are y'all in?

**Reply by Flutterboy17** [reply]
VS Code

**Reply by ayim** [reply]
@Akash1134

**Reply by zjx00zjxlive** [reply]
pycharm , vscode , and android studio.

**Comment by davecheney** [comment, marked as answer]
@zjx00zjxlive please contact support, we cannot resolve billing issues via this forum. Thank you.

@Flutterboy17 same story.
